src/services/qrProcessor/qrProcessor.test.js

```diff
@@ -1,39 +1,64 @@
 import { decodeQrCode, processQrCode } from "./index";
 
 const encodeQrCode = (payload) => `https://action.openattestation.com?q=${encodeURIComponent(JSON.stringify(payload))}`;
```

Comment:
should this sit at index too? then import from there

Reply:
It can be moved to index but `encodeQrCode()` is not used in the project. Only a helper function for the purposes of testing.